cluster/ci/config/prow/plugins.yaml

```diff
@@ -1094,3 +1097,25 @@ external_plugins:
     - name: needs-rebase
       events:
       - pull_request
+
+bugzilla:
+  default:
+    openshift-4.0:
+      is_open: true
+      target_release: 4.0.0
+    release-4.0:
+      is_open: true
+      target_release: 4.0.0
+    openshift-4.1:
+      is_open: true
+      target_release: 4.1.0
+    release-4.1:
+      is_open: true
+      target_release: 4.1.0
```

Comment:
Target release is what the bug has to be opened against as a target?  If so, this should be `4.1.z` I think.

Reply:
done